.vimrc

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Never mind, I see in the description that this was intentional. In the future, we'll probably use something like [Prettier](https://prettier.io/) to enforce this formatting.

Reply:
Yeah this is really just for vim only, but since the server and client have different indentations already seemed the easiest way to make my development sane.